assets/wizards/popups/index.js

```diff
@@ -224,14 +198,64 @@ class PopupsWizard extends Component {
 		return `${ previewURL }?${ stringify( { ...options, newspack_popups_preview_id: id } ) }`;
 	};
 
+	updateAfterAPI = ( { campaigns, prompts, segments, settings } ) =>
+		this.setState( { campaigns, prompts, segments, settings } );
+
 	manageCampaignGroup = ( campaigns, method = 'POST' ) => {
 		const { setError, wizardApiFetch } = this.props;
 		return wizardApiFetch( {
 			path: '/newspack/v1/wizard/newspack-popups-wizard/batch-publish/',
 			data: { ids: campaigns.map( campaign => campaign.id ) },
 			method,
+			quiet: true,
+		} )
+			.then( this.updateAfterAPI )
+			.catch( error => setError( error ) );
+	};
+
+	renameCampaignGroup = ( id, name ) => {
+		const { setError, wizardApiFetch } = this.props;
+		return wizardApiFetch( {
+			path: `/newspack/v1/wizard/newspack-popups-wizard/rename-campaign/${ id }`,
+			method: 'POST',
+			data: { name },
+			quiet: true,
 		} )
-			.then( () => this.onWizardReady() )
+			.then( this.updateAfterAPI )
+			.catch( error => setError( error ) );
+	};
+
+	archiveCampaignGroup = ( id, status ) => {
```

Comment:
Feels a bit weird to me that archiving a campaign with active prompts won't deactivate the prompts. I get why that is (because the prompts might belong to other campaigns as well). But in many cases prompts will only belong to one campaign. Maybe we can ask the user with a modal if they want to also deactivate the prompts in the campaign? Might also be helpful when deleting a campaign.

Reply:
I agree it feels weird, but I think this will require discussion. We could:

1) Unpublish all prompts within the campaign.
2) Unpublish all prompts that are in the campaign and are in no others.
3) If prompts are found in current and other campaigns, ask whether to unpublish them.
4) Do nothing (status quo).

None are perfect - what do you think?

Reply:
(3) feels like the best balance to me, but probably requires the most effort and might be beyond this PR's scope. Let's table this discussion for now and get BB's opinion for another potential PR.

Reply:
cc: @brianboyer

Reply:
I agree that there is no perfect solution. To be honest, archiving campaigns makes a little less sense to me in a system where prompts aren't contained within a campaign.

We already have the ability to unpublish all prompts in the campaign, so I imagine the user's workflow would be: 1. Unpublish the prompts. 2. Archive the campaign.

Let's try option 4 and if it doesn't make sense, consider other paths?